tools/csv-merger/csv-merger.go

```diff
@@ -83,7 +83,9 @@ var (
 	namespace           = flag.String("namespace", "kubevirt-hyperconverged", "Namespace")
 	crdDisplay          = flag.String("crd-display", "KubeVirt HyperConverged Cluster", "Label show in OLM UI about the primary CRD")
 	csvOverrides        = flag.String("csv-overrides", "", "CSV like string with punctual changes that will be recursively applied (if possible)")
-	relatedImagesList   = flag.String("related-images-list", "",
+	visibleCRDList      = flag.String("visible-crds-list", "hyperconvergeds.hco.kubevirt.io,v2vvmwares.kubevirt.io,hostpathprovisioners.hostpathprovisioner.kubevirt.io",
```

Comment:
my only question is whether or not this needs to be a command line argument. It looks to me like the kind of thing we would always change in the source.

Here is my thought, specifically create vars for visible and hidden CRDs, if we see one that isn't in either `panic` because we missed something. WDYT?

Reply:
I added it as CLI argument (with a reasonable default) because I cannot exclude we can have a different configuration between upstream and downstream